lib/services/database.go

```diff
@@ -34,6 +35,7 @@ import (
 	"github.com/aws/aws-sdk-go/service/redshift"
 	"github.com/coreos/go-semver/semver"
 	"github.com/gravitational/trace"
+	"github.com/sirupsen/logrus"
```

Comment:
nit: The package is already imported below as `log "github.com/sirupsen/logrus"`